app/views/organizations/adopter_fosterer/fostered_pets/index.html.erb

```diff
@@ -4,7 +4,7 @@
     <div data-controller="card">
       <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 g-4 mb-5">
         <% @fostered_pets.each_with_index do |fostered_pet, index| %>
-          <div class="col">
+          <div class="col-lg-5 col-md-6">
```

Comment:
Instead of using grid here, I think we can simplify using flex on each Pet card?